lib/Dialect/AIE/Transforms/AIECanonicalizeDevice.cpp

```diff
@@ -27,9 +27,9 @@ struct AIECanonicalizeDevicePass
 
     ModuleOp moduleOp = getOperation();
 
-    for (auto deviceOp : moduleOp.getOps<AIETarget>()) {
+    for (auto deviceOp : moduleOp.getOps<DeviceOp>()) {
```

Comment:
Why did you need this?

Reply:
I removed that interface - there was only one op that implemented it `AIEDeviceOp`